src/FlaUI.Core/FlaUI.Core.csproj

```diff
@@ -1,15 +1,17 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
-
+﻿<Project>
+  <Sdk Condition="'$(TargetFramework)' != 'net5.0-windows'" Name="Microsoft.NET.Sdk.WindowsDesktop" />
```

Comment:
Unfortunately this does not work as this adds the imports before everything else. Fortunately it is allowed to use `Microsoft.NET.Sdk.WindowsDesktop` also in .net 5 and there is no warning shown anymore. See https://github.com/dotnet/sdk/pull/13441